### ISSUE TITLE
Extension theme configuration

### ISSUE THREAD

**Issue by hinell** [benfowler/telescope-luasnip.nvim#16]
```lua
require("telescope").setup({
	config = function()
		require("telescope").setup({
			...
			extensions = {
				luasnip = {
					theme = "dropdown" 
				}
			}
		})
	end
})
```

The above config doesn't work. I'm surprised why.

**Comment by benfowler** [comment]
Hi @hinell ! In what way is this breaking for you? Do you have a minimal Neovim configuration that reproduces the breakage you're seeing?

**Comment by hinell** [comment]
@benfowler This is the minimal config I have. There is nothing that affects this plugin. I took a look at the this repo's code and it seems like you don't handle opts in `setup(...)`.

**Comment by filipgodlewski** [comment]
Neither does work for me, but @hinell, as a workaround you can use:
```lua
vim.cmd "Telescope luasnip theme=dropdown"
```
and it will work just fine.

EDIT:
or you can use this:
```lua
require("telescope").extensions.luasnip.luasnip(
  require"telescope.themes".get_dropdown { layout_config = { width = 0.5 } }
)
```

**Comment by hinell** [comment]
@filipgodlewski Yeah, I know about that. Thanks!

**Comment by benfowler** [comment]
@hinell I think your recent PR addressed this configuration issue by reworking extension configuration to work the proper way? Please correct me if I'm wrong. If so, we can close this issue.

**Comment by hinell** [comment]
Fixed by #19

@benfowler Thanks for speedy merge!
